src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,86 @@
+.product-card {
+  box-sizing: border-box;
+  height: 406px;
+  width: 200px;
+  padding: 206px 16px 16px;
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: 19px 32px;
+  border: 1px solid#f3f3f3;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), hex color values should be lowercase and properly formatted. Use `#f3f3f3` instead of `#f3f3f3`.